portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Comments</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+  </head>
+  <body onload="showComments()">
+    <div id="barTop">
+      <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #004047">
+        <a class="navbar-brand" href="index.html">Andrés's Portfolio</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" 
+            aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+          <a class="nav-item nav-link" href="index.html">Home</a>
+          <a class="nav-item nav-link" href="hobbies.html">Hobbies</a>
+          <a class="nav-item nav-link active" href="#">Comments<span class="sr-only">(current)</span></a>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div id="content">
+      <div id="title">
+        <h1>Comments</h1>
+      </div>
+      <form action="/data" method="POST">
+        <!-- TODO(aabundis): Add pseudonym feature to identify owner of comment. -->
+        <div class="form-group">
+          <label for="textarea"><h3 class="subtitle">Leave your comment:</h3></label>
+          <textarea class="form-control" name="textarea" rows="3"></textarea>
+        </div>
+        <button type="submit" class="btn btn-light">Submit</button>
+      </form>
+      <h3 class="subtitle">Past comments:</h3>
+      <!-- Container used to display the returned list children from showContent() -->
+      <div>
+        <ol id="content-container"></ol>
+      </div>
+    </div>
+  </body>
+</html>
+ 
```

Comment:
Hm for some reason Github is saying that there's no newline at the end of the file but it looks like there is...maybe just double check that a newline does exist?

Reply:
There was an extra space in the last line so perhaps that was causing Github to confuse.